api/v1beta1/conversion_test.go

```diff
@@ -76,20 +76,20 @@ var _ = Describe("Conversion", func() {
 		})
 	})
 
-	Context("v1beta2 to v1beta1 function", func() {
```

Comment:
Do we need corresponding beta3 to beta2 tests given that's still served.

Reply:
Yep we will need them. All tests unit and e2e, & webhook changes are coming up in next couple PRs